packages/devtools_app/lib/src/shared/ui/gtags.dart

```diff
@@ -22,7 +22,7 @@ import '../analytics/analytics.dart' as ga;
 bool _debugAnalytics = false;
 
 @JS('gtag')
-external void _gTagCommandName(String command, String name, [dynamic params]);
+external void _gTagCommandName(String command, String name, [Object params]);
```

Comment:
should this be `Object?` - here and below

Reply:
Done. Here and elsewhere made a lot more cases of `dynamic` into `Object?` instead of `Object`